install/operator/pkg/cmd/internal/grant/grant.go

```diff
@@ -75,8 +78,16 @@ func (o *Grant) grant() error {
 	if err != nil {
 		return err
 	}
-
 	resources = append(resources, gr...)
+
+	if o.publicApi {
+		pubRole, err := generator.Render("./install/grant_public_api_cluster_role.yml.tmpl", o)
+		if err != nil {
+			return err
+		}
+		resources = append(resources, pubRole...)
+	}
+
```

Comment:
I would rather have this snippet in `pkg/syndesis/action/install`, something like.

```
	if configuration.Syndesis.Addons.PublicApi.Enabled  {
		pubRole, err := generator.Render("./install/grant_public_api_cluster_role.yml.tmpl", o)
		if err != nil {
			return err
		}
		resources = append(resources, pubRole...)
	}
```